pkg/config/apm.go

```diff
@@ -110,10 +110,26 @@ func setupAPM(config Config) {
 	})
 
 	config.SetEnvKeyTransformer("apm_config.filter_tags.require", func(in string) interface{} {
```

Comment:
Perhaps you could make a helper function out of it.
```suggestion
	config.SetEnvKeyTransformer("apm_config.filter_tags.require", parseKVList(""apm_config.filter_tags.require""))
```